... (#2885)
ansys/pyfluent: doc/source/user_guide/tui_commands.rst

   fluent.exe 3ddp -i my_journal.jou -topy

In Fluent 2023 R1, any call to a TUI command, for which a settings API analogue exists,
is recorded as a settings API call in the Python journal. Otherwise, a call to 

> ``execute_tui(<argument>)`` is recorded and ``<argument>`` is a string representing the 

Reviewer: ```suggestion
``execute_tui(<argument>)`` is recorded, where ``<argument>`` is a string representing the 
```

original TUI call. You need to manually convert those TUI commands using the transformation
rules described in the next section.

> In Fluent 2022 R2, the Python journaling feature is not available at all, so uou need to manually

Reviewer: ```suggestion
In Fluent 2022 R2, the Python journaling feature is not available at all, so you must manually
```
